WooCommerce/Classes/ViewRelated/Products/Media/ProductImageStatus.swift

```diff
@@ -24,6 +24,19 @@ extension Collection where Element == ProductImageStatus {
             }
         }
     }
+
+    /// Whether there are still any images being uploaded.
+    ///
+    var hasImagesPendingUpload: Bool {
```

Comment:
I believe there's enough context to just say `hasPendingUpload`. What do you think? 🙂